datahub-web-react/src/app/entity/shared/tabs/Dataset/AccessManagement/AccessRolesDetail.tsx

```diff
@@ -0,0 +1,27 @@
+export default function handleExternalRoles(externalRoles, loggedInUser) {
```

Comment:
let's name this file utils.ts since it's just a utility function

we can also remove the "Default" in front of function

Reply:
renamed to utils.ts